browser/extensions/BUILD.gn

```diff
@@ -71,7 +70,8 @@ source_set("extensions") {
     "//brave/components/decentralized_dns/buildflags",
     "//brave/components/ipfs/buildflags",
     "//brave/components/sidebar/buildflags",
-    "//brave/components/speedreader",
+    "//brave/components/speedreader/buildflags",
+    "//brave/components/speedreader/common",
```

Comment:
you should add this dep if speedreader is enabled, to do that you'd need to import buildflags and check the flag

Reply:
fixed